doc/cephadm/services/rgw.rst

```diff
@@ -74,6 +74,33 @@ example spec file:
     spec:
       rgw_frontend_port: 8080
 
+Passing Frontend Extra Arguments
+--------------------------------
+
+The RGW service specification can be used to pass extra arguments to the rgw frontend by using
```

Comment:
s/rgw/RGW/

Reply:
While this is true, this is a quincy backport so if we'd like to change it we should probably have a patch in main and then propagate it down. Same goes for the other comment.